apps/front-end/src/attributes/attributes.component.ts

```diff
@@ -1,24 +1,48 @@
-import { Component } from "@angular/core";
+import { Component, inject, OnInit } from "@angular/core";
 import { TableComponent } from "../shared/table.component";
 import { Columns } from "../shared/table.component.types";
+import { MatTableDataSource } from "@angular/material/table";
+import { Subject, takeUntil } from "rxjs";
+import { AttributesService } from "../services/attributes.service";
 
-@Component({ 
+@Component({
     imports: [TableComponent],
     selector: 'app-attributes',
     templateUrl: './attributes.component.html',
 })
 
-export class AttributesComponent {
+export class AttributesComponent implements OnInit {
+    private attributesService = inject(AttributesService);
+    private destroy$ = new Subject<void>();
```

Comment:
Remove this since its not being used. Ideally it would be fired on the ngOnDestroy lifecycle hook. You can use takeUntilDestroyed() operator instead on the subscription